WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
I’m new to this stuff so bear with me: tried just dropping a warning inside the if statement but it gets returned twice. Can’t seem to figure out why that is happening :-(

```
if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
	$message = 'The \'tax_query\' whitelist flag is deprecated in favor of \'slow query\'.';
	$this->addMessage( $message, $stackPtr, false );
	return;
}
```